### ISSUE TITLE
Release drafter removes words inside back tick quotes when creating a PR

### ISSUE THREAD

**Issue by obulat** [WordPress/openverse#1189]
## Description

The [PR](https://github.com/WordPress/openverse/pull/1185/files) that would publish the API changelog removes all the words that were in back tick quotes. The [GitHub version](https://github.com/WordPress/openverse/releases/tag/api-2023.04.12.23.29.59), interestingly, is not affected.

## Reproduction

See, for example, line 15 here: https://github.com/WordPress/openverse/pull/1186/files

## Screenshots

`- Absorb `build-nginx` job into `build-images` job (#944) @dhruvkb` becomes `- Absorb job into job (#944) @dhruvkb`

## Additional context

It seems that we could use [replacers](https://github.com/release-drafter/release-drafter#replacers) to fix this issue.
One more thing that we might want to fix when publishing the changelogs to docs.openverse.org would be the links to the PRs. GitHub automatically inserts them when it sees the PR numbers, but our docs site won't.

**Comment by AetherUnbound** [comment]
This _should_ already be done by the `link_issues` Sphinx plugin we added here 😮 https://github.com/WordPress/openverse/pull/987#discussion_r1147846444

https://github.com/WordPress/openverse/blob/777d087c3cb6b3f2416a126f0b5cfcb121531404/documentation/conf.py#L56-L57

That must not be working properly, so we'll need to look into why.

**Comment by obulat** [comment]
Oh nice!
Is the bug with the back ticks also solved?

**Comment by AetherUnbound** [comment]
Sorry, my comments above were only for your extra note! On the backtics thing though, I think I know what's going on!

https://github.com/WordPress/openverse/blob/09d6bf48b1713988ef3ccfb72a0c47268af49785/.github/workflows/release-app.yml#L113-L117

The backtics in this line must be being interpreted by bash as interpolated commands 😮 That's why we're seeing all these weird errors on the "Add new changelog file to documentation" step of this run: https://github.com/WordPress/openverse/actions/runs/4683846102/jobs/8299355516

```
Run cat << EOF > documentation/changelogs/api/2023.04.12.23.29.59.md
  cat << EOF > documentation/changelogs/api/2023.04.12.23.29.59.md
  # 2023.04.12.23.29.59
  
  ## New Features
  
  - Add a Slack notification job to the CI + CD workflow (#1066) @dhruvkb
  - Set rate-limit headers on each response (#775) @dhruvkb
  
  ## Improvements
  
  - Fix issues in the workflow simplifications of #1054 (#1058) @dhruvkb
  - Simplify CI + CD workflow (#1054) @dhruvkb
  - Improve documentation for partial stack setups (#974) @dhruvkb
  -  Use upstream thumbnail if available (#898) @krysal
  - Remove XML from the API (#986) @obulat
  - Update URLs to point to docs.openverse.org (#991) @dhruvkb
  - Absorb `build-nginx` job into `build-images` job (#944) @dhruvkb
  
  ## Internal Improvements
  
  - Change deployment workflow name from colon to dash (#1174) @AetherUnbound
  - Safely call create-or-update-comment when dealing with forks (#997) @sarayourfriend
  - Update nginx Docker tag to v1.23.4 (#1108) @renovate
  - Use context manager for multiprocessing in the ingestion server (#1057) @obulat
  - Bump boto3 from 1.26.99 to 1.26.105 in /api (#1133) @dependabot
  - Prepare VButton for updates (#1002) @obulat
  - Add `SLACK_WEBHOOK_TYPE` env var to reporting job (#1131) @dhruvkb
  - Fix types in VFilters and VContentReport (#1030) @obulat
  - Add a Slack notification job to the CI + CD workflow (#1066) @dhruvkb
  - Bump pillow from 9.4.0 to 9.5.0 in /api (#1115) @dependabot
  - Bump sentry-sdk from 1.17.0 to 1.18.0 in /api (#1112) @dependabot
  - Bump orjson from 3.8.8 to 3.8.9 in /api (#1114) @dependabot
  - Bump ipython from 8.11.0 to 8.12.0 in /api (#1113) @dependabot
  - Pass actor for staging deploys with the `-f` flag (#1104) @dhruvkb
  - Dispatch workflows instead of regular reuse to show deployment runs (#1034) @sarayourfriend
  - Restore Django Admin views (#1065) @krysal
  - Save cleaned up data during the cleanup step (#904) @obulat
  - Defer the `tags_list` for media models (#1029) @obulat
  - Bump boto3 from 1.26.97 to 1.26.99 in /api (#1042) @dependabot
  - Add tag app release action (#987) @sarayourfriend
  - Add CNAME in other use of `actions-gh-pages` (#1006) @dhruvkb
  - Fix local build of the API and add its `recreate` just command (#994) @krysal
  - Skip build and publish job if nothing to do (#977) @dhruvkb
  
  ## Bug Fixes
  
  - Change deployment workflow name from colon to dash (#1174) @AetherUnbound
  - Safely call create-or-update-comment when dealing with forks (#997) @sarayourfriend
  - Pass `GITHUB_TOKEN` to deploy docs (#1134) @dhruvkb
  - Add `SLACK_WEBHOOK_TYPE` env var to reporting job (#1131) @dhruvkb
  - Fix types in VFilters and VContentReport (#1030) @obulat
  - Pass actor for staging deploys with the `-f` flag (#1104) @dhruvkb
  - Add `GITHUB_TOKEN` to GitHub CLI step (#1103) @dhruvkb
  - Dispatch workflows instead of regular reuse to show deployment runs (#1034) @sarayourfriend
  - Fix typo in docs building on `main` (#1067) @dhruvkb
  - Add tag app release action (#987) @sarayourfriend
  - Add CNAME in other use of `actions-gh-pages` (#1006) @dhruvkb
  - Add docs CNAME to config (#1005) @zackkrida
  - Fix diagrams with transparent background in README.md of "ingestion_server" for dark mode. (#1000) @AdarshRawat1
  - Fix local build of the API and add its `recreate` just command (#994) @krysal
  - Treat any non 200 status as failure for thingiverse (#940) @sarayourfriend
  - Skip build and publish job if nothing to do (#977) @dhruvkb
  
  ## Credits
  
  Thanks to @AdarshRawat1, @AetherUnbound, @Tomvth, @dependabot, @dependabot[bot], @dhruvkb, @glowatsk, @kk311y, @krysal, @obulat, @openverse-bot, @panchovm, @raiyaj, @renovate, @renovate[bot], @sarayourfriend, @sepehrrezaei, @sumit-158 and @zackkrida for their contributions!
  
  EOF
  shell: /usr/bin/bash -e {0}
/home/runner/work/_temp/36d15401-d314-444e-a1e5-dff31f656e27.sh: line 1: build-nginx: command not found
/home/runner/work/_temp/36d15401-d314-444e-a1e5-dff31f656e27.sh: line 1: build-images: command not found
/home/runner/work/_temp/36d15401-d314-444e-a1e5-dff31f656e27.sh: line 1: SLACK_WEBHOOK_TYPE: command not found
/home/runner/work/_temp/36d15401-d314-444e-a1e5-dff31f656e27.sh: line 1: -f: command not found
/home/runner/work/_temp/36d15401-d314-444e-a1e5-dff31f656e27.sh: line 1: tags_list: command not found
/home/runner/work/_temp/36d15401-d314-444e-a1e5-dff31f656e27.sh: line 1: actions-gh-pages: command not found
/home/runner/work/_temp/36d15401-d314-444e-a1e5-dff31f656e27.sh: line 1: recreate: command not found
/home/runner/work/_temp/36d15401-d314-444e-a1e5-dff31f656e27.sh: line 1: GITHUB_TOKEN: command not found
/home/runner/work/_temp/36d15401-d314-444e-a1e5-dff31f656e27.sh: line 1: SLACK_WEBHOOK_TYPE: command not found
/home/runner/work/_temp/36d15401-d314-444e-a1e5-dff31f656e27.sh: line 1: -f: command not found
/home/runner/work/_temp/36d15401-d314-444e-a1e5-dff31f656e27.sh: line 1: GITHUB_TOKEN: command not found
/home/runner/work/_temp/36d15401-d314-444e-a1e5-dff31f656e27.sh: line 1: main: command not found
/home/runner/work/_temp/36d15401-d314-444e-a1e5-dff31f656e27.sh: line 1: actions-gh-pages: command not found
/home/runner/work/_temp/36d15401-d314-444e-a1e5-dff31f656e27.sh: line 1: recreate: command not found
```

We'll have to find a way to output the contents to a file without having bash interpret the contents within the backtics.
